home-service/src/hooks/useBookingRequest.ts

```diff
@@ -0,0 +1,26 @@
+
+import { UseMutationResult, useMutation, useQueryClient } from '@tanstack/react-query';
+import { ErrorResponse } from 'react-router-dom';
+import { BookingRequest } from '@/types/booking';
+import { postBooking } from '@/api/bookingApi';
+import useLocalStorage from './useLocalStorage';
+
+export const BOOKINGS_KEY = 'BOOKINGS'
+
+export const useBookingRequest = (): UseMutationResult<void, ErrorResponse, BookingRequest, unknown> => {
+  const queryClient = useQueryClient();
+  const [token] = useLocalStorage<string | null>('token', null);
```

Comment:
šitą į instance reikia persikelti, nes kas kartą bus negerai

Reply:
fixed